guides/doc-Release_Notes/master.adoc

```diff
@@ -15,6 +15,9 @@ endif::[]
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.7.0.adoc[leveloffset=+1]
 include::topics/katello-4.9.0.adoc[leveloffset=+1]
+include::topics/katello-4.9.1.adoc[leveloffset=+1]
+include::topics/katello-4.9.2.adoc[leveloffset=+1]
```

Comment:
We order them from oldest to newest. You should also include `ifdef::katello[]` above it and `endif::[]` after. See the 3.6 branch for an example: https://github.com/theforeman/foreman-documentation/blob/3.6/guides/doc-Release_Notes/master.adoc?plain=1

Reply:
OK, I reversed the order and wrapped it in a conditional